cfme/tests/intelligence/test_reports_with_timelines.py

```diff
@@ -1,39 +1,144 @@
+        on the CloudIntel->Timelines page
 
-@pytest.mark.manual
-@pytest.mark.tier(2)
-@pytest.mark.parametrize('report', timelines_reports)
-def test_custom_reports_with_timelines(report):
+        Args:
+            request: py.test funcarg request
+            appliance: IPAppliance object
+            provider: provider object
+            path: path to the report in the tree
+            updates: updates to the default report so that it will display events
+            vm_crud: vm for Policy events
+            register_event: function for registering events
+        """
+        # first perform extra steps for Policy stuff
+        if "Policy" in path:
+            generate_policy_event(request, appliance, provider, vm_crud, register_event)
+        report = appliance.collections.reports.instantiate(
+            type=path[0], subtype=path[1], menu_name=path[2]
+        )
+        # copy the default report
+        copied_report = report.copy()
+        # update the copied report
+        copied_report.update(updates)
+        # delete report at end of test
+        request.addfinalizer(copied_report.delete)
+
+        return copied_report
+
+    return _setup_for_reports
+
+
+@pytest.fixture(scope="function")
+def vm_crud(provider, setup_provider, small_template):
+    template = small_template
+    # GCE providers require the '-' instead of '_' to work properly with cleanup
+    base_name = "test-events-{}" if provider.one_of(GCEProvider) else "test_events_{}"
+    vm_name = base_name.format(fauxfactory.gen_alpha(length=8).lower())
+
+    collection = provider.appliance.provider_based_collection(provider)
+    vm = collection.instantiate(vm_name, provider, template_name=template.name)
+    yield vm
+    vm.cleanup_on_provider()
+
+
+def generate_policy_event(request, appliance, provider, vm_crud, register_event):
+    # create necessary objects
+    action = appliance.collections.actions.create(
+        fauxfactory.gen_alpha(), "Tag", dict(tag=("My Company Tags", "Environment", "Development"))
+    )
+    request.addfinalizer(action.delete)
+
+    policy = appliance.collections.policies.create(
+        policies.VMControlPolicy, fauxfactory.gen_alpha()
+    )
+    request.addfinalizer(policy.delete)
+
+    policy.assign_events("VM Create Complete")
+    request.addfinalizer(policy.assign_events)
+    policy.assign_actions_to_event("VM Create Complete", action)
+
+    profile = appliance.collections.policy_profiles.create(
+        fauxfactory.gen_alpha(), policies=[policy]
+    )
+    request.addfinalizer(profile.delete)
+
+    # assign the policy profile to the provider
+    provider.assign_policy_profiles(profile.description)
+    request.addfinalizer(lambda: provider.unassign_policy_profiles(profile.description))
+
+    register_event(target_type="VmOrTemplate", target_name=vm_crud.name, event_type="vm_create")
+
+    vm_crud.create_on_provider(find_in_cfme=True)
+
+
+@pytest.mark.parametrize("path, updates", zip(PATHS, UPDATES))
```

Comment:
**Optional**

I think it would be better to define your parametrization pairs together, as a giant list of tuples. Using namedtuple will allow you to reference them consistently.

** Required **
Pass `ids`  for this parametrization - currently you're getting test functions of: `test_reports_with_timelines[virtualcenter-6.0-path0-updates0]`